src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_PropertyAccess.cs

```diff
@@ -67,19 +67,21 @@ private BoundExpression VisitPropertyAccess(BoundPropertyAccess node, bool isLef
 
         private BoundExpression MakePropertyGetAccess(SyntaxNode syntax, BoundExpression? rewrittenReceiver, PropertySymbol property, BoundPropertyAccess? oldNodeOpt)
         {
-            return MakePropertyGetAccess(syntax, rewrittenReceiver, property, ImmutableArray<BoundExpression>.Empty, null, oldNodeOpt);
+            return MakePropertyGetAccess(syntax, rewrittenReceiver, property, ImmutableArray<BoundExpression>.Empty, default, null, oldNodeOpt);
         }
 
         private BoundExpression MakePropertyGetAccess(
             SyntaxNode syntax,
             BoundExpression? rewrittenReceiver,
             PropertySymbol property,
             ImmutableArray<BoundExpression> rewrittenArguments,
+            ImmutableArray<RefKind> argumentRefKindsOpt,
             MethodSymbol? getMethodOpt = null,
             BoundPropertyAccess? oldNodeOpt = null)
         {
             if (_inExpressionLambda && rewrittenArguments.IsEmpty)
             {
+                Debug.Assert(argumentRefKindsOpt.IsDefault);
```

Comment:
>Debug.Assert(argumentRefKindsOpt.IsDefault); [](http://example.com/codeflow?start=16&length=44)

It is not obvious why this assumption is justified.

Reply:
Because `rewrittenArguments` are empty, so I think their ref kinds must be also empty - and in fact default, because they won't be even allocated; but I can relax (to IsDefaultOrEmpty) or remove this assert if you prefer.

Reply:
>but I can relax (to IsDefaultOrEmpty) or remove this assert if you prefer.

Let's do one or the other.